wordpress-mobile/WordPress-iOS
# Reader site filter: show unseen post count

Ref #15483

On the Reader's site filter, a unseen post count is now displayed for each site.

To test:
- Go to the Reader.
- On any feed that has a filter (Following, Automattic, P2s), tap `Filter`.
- Verify:
  - For sites that have unseen posts, the count is displayed.
  - For sites that have no unseen posts, the count is not displayed.
  - The counts match those in the web Reader.
  - Numbers above 1000 are abbreviated with multipliers(K, M, B, etc.)

To verify large numbers are abbreviated, you can hack the displayed count. In `FilterProvider:addUnseenPostCount`, change the displayed value: `countLabel.text = topic.unseenCount.abbreviatedString()`

Example:
```
let count = 876875666978
countLabel.text = count.abbreviatedString()
```

---
| ![unseen_count](https://user-images.githubusercontent.com/1816888/103695045-22d4ff00-4f59-11eb-8b34-ed43d9422338.png) | ![abbrevs](https://user-images.githubusercontent.com/1816888/103695068-28cae000-4f59-11eb-9042-6ed838d2f25f.png) |
|--------|-------|

| ![ipad_light](https://user-images.githubusercontent.com/1816888/103695281-83643c00-4f59-11eb-933e-ca83a738cc73.png) | ![ipad_dark](https://user-images.githubusercontent.com/1816888/103695305-8d863a80-4f59-11eb-8efe-af9bc021b107.png) |
|--------|-------|

---
PR submission checklist:
